[2022-07-27 19:45:08,840] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dataquality_pipeline_v3.upload_to_s3_2020-05 manual__2022-07-27T19:44:56.114008+00:00 [queued]>
[2022-07-27 19:45:08,849] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dataquality_pipeline_v3.upload_to_s3_2020-05 manual__2022-07-27T19:44:56.114008+00:00 [queued]>
[2022-07-27 19:45:08,850] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-27 19:45:08,850] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-07-27 19:45:08,850] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-27 19:45:08,865] {taskinstance.py:1397} INFO - Executing <Task(LocalFilesystemToS3Operator): upload_to_s3_2020-05> on 2022-07-27 19:44:56.114008+00:00
[2022-07-27 19:45:08,877] {standard_task_runner.py:52} INFO - Started process 10570 to run task
[2022-07-27 19:45:08,895] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dataquality_pipeline_v3', 'upload_to_s3_2020-05', 'manual__2022-07-27T19:44:56.114008+00:00', '--job-id', '165', '--raw', '--subdir', 'DAGS_FOLDER/dataquality_pipeline_v3.py', '--cfg-path', '/tmp/tmpts71qrrs', '--error-file', '/tmp/tmpcgxy_7mw']
[2022-07-27 19:45:08,902] {standard_task_runner.py:80} INFO - Job 165: Subtask upload_to_s3_2020-05
[2022-07-27 19:45:09,018] {task_command.py:371} INFO - Running <TaskInstance: dataquality_pipeline_v3.upload_to_s3_2020-05 manual__2022-07-27T19:44:56.114008+00:00 [running]> on host 2c2de32fb47f
[2022-07-27 19:45:09,240] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=abin404@yopmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dataquality_pipeline_v3
AIRFLOW_CTX_TASK_ID=upload_to_s3_2020-05
AIRFLOW_CTX_EXECUTION_DATE=2022-07-27T19:44:56.114008+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-07-27T19:44:56.114008+00:00
[2022-07-27 19:45:09,270] {base.py:68} INFO - Using connection ID 'aws-conn' for task execution.
[2022-07-27 19:45:09,272] {base_aws.py:206} INFO - Credentials retrieved from login
[2022-07-27 19:45:10,441] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=dataquality_pipeline_v3, task_id=upload_to_s3_2020-05, execution_date=20220727T194456, start_date=20220727T194508, end_date=20220727T194510
[2022-07-27 19:45:10,493] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-27 19:45:10,653] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
